sklearn/utils/estimator_checks.py

```diff
@@ -1191,6 +1192,57 @@ def check_estimators_pickle(name, estimator_orig):
         assert_allclose_dense_sparse(result[method], unpickled_result)
 
 
+def check_pairwise_estimator_tag(name, estimator_orig):
+    attributes_to_check = ['metric', 'affinity', 'kernel']
+    attribute_value = 'precomputed'
```

Comment:
There's no need to make a variable for this. It just makes it hard to read. Please just put a literal string in line

Reply:
@jnothman sorry for the delay! fixed